tests/phpunit/tests/cache.php

```diff
@@ -13,11 +13,6 @@ function setUp() {
 		$this->cache =& $this->init_cache();
 	}
 
-	function tearDown() {
-		$this->flush_cache();
```

Comment:
Cache is flushed between test classes via `tearDownAfterClass()`, but as this class is specifically testing the cache, I suspect this call to `flush_cache()` _may_ have a function here.

Reply:
I did consider this but the cache is flushed before every test too, via `setUp()` -> `clean_up_global_scope()` -> `flush_cache()`.